doc/plotting.rst

```diff
@@ -41,7 +41,7 @@ Rule ``plot_p_nom_max``
 
 |
 
-.. automodule:: plot_p_nom_max
+.. .. automodule:: plot_p_nom_max
```

Comment:
Is this double double dot needed?

Reply:
the extra double dot is used to comment the section out. I didn't want to remove it, that's why I applied the comment syntax.